webpack-modules/README.md

```diff
@@ -0,0 +1,31 @@
+# WebExtension Webpack Example
+A minimal example of how to use npm modules from within a WebExtension.
+The example package used by this extension is `left-pad`, an essential package
+in almost any situation.
+
+## What it does
+This setup takes a background and a content script entrypoint and outputs them
```

Comment:
I think "left-pad.js" is not a content script is it? It's a popup script (meaning, it runs in the chrome process, can get direct access to the background page, and can access all the privileged APIs that the background page can).

This makes it a bit strange that the popup has the option of sending a message to the background script to do the padding. I think it would be better to omit the background script from this example, and just demonstrate using the popup script. Unless there's some reason I'm missing, that you are including both?